sway/desktop/fx_renderer/fx_renderer.c

```diff
@@ -760,17 +792,10 @@ void fx_render_box_shadow(struct fx_renderer *renderer, const struct wlr_box *bo
 	glClearStencil(0);
 	glClear(GL_STENCIL_BUFFER_BIT);
 
```

Comment:
Looks like this would be handled in your `fx_renderer_stencil_mask_init();` function. Would we want to add `glEnable(GL_STENCIL_TEST);` to that function as well?